csfieldguide/static/interactives/binary-cards/README.md

```diff
@@ -9,9 +10,10 @@ This interactive is created to teach binary numbers, and emulates the Binary Car
 The interactive shows the cards with dots from **128** to **1** by default (128, 64, 32, 16, 8, 4, 2, 1), however these can be configured by the following parameters:
 
 - `base=value` - Where `value` is the number base to use (defaults as 2).
-- `digits=value` - Where `value` is the amounts of digits to display (default is 6).
+- `digits=value` - Where `value` is the amount of digits to display (default is 8). Min is 1 and max is 16.
```

Comment:
Change to `number` rather than `amount`
An aside, but I still don't understand why it's called digits. I guess they're 'binary digits' but since that's the only reference to binary in the interactive, 'cards' seems more appropriate